cpu/nrf52/Makefile.include

```diff
@@ -1,6 +1,10 @@
 export CPU_ARCH = cortex-m4f
 export CPU_FAM  = nrf52
 
+# Bootloader length needs to be 8k so slot0 & slot1 start from the beegining
```

Comment:
Typo: beegining -> beginning.

Maybe shortly state sth like, ```Slot size is determined by "((total_flash_size - RIOTBOOT_LEN) / 2)". If RIOTBOOT_LEN uses an uneven number of flashpages, the remainder of the flash is cannot be divided by two slots while staying FLASHPAGE_SIZE aligned.```

Reply:
Agreed and addressed.